lib/rspec/core/pending.rb

```diff
@@ -52,21 +52,42 @@ class PendingExampleFixedError < StandardError; end
       #       end
       #     end
       #
-      # @note `before(:example)` hooks are eval'd when you use the `pending`
-      #   method within an example. If you want to declare an example `pending`
-      #   and bypass the `before` hooks as well, you can pass `:pending => true`
-      #   to the `it` method:
+      # @example Alternatively, you can use the `pending` example alias method:
```

Comment:
This doc is still upside down, it is the documentation for the method pending, not the metadata, the caveat about hooks is more important that documenting the metadata functionality which is documented elsewhere, metadata is only relevant as an alternative which fixes this.

Given how this renders, (e.g. note is on top) it needs to be much shorter.

![Screen Shot 2021-02-24 at 10 31 15](https://user-images.githubusercontent.com/162976/108987700-7deabe80-768b-11eb-9b40-4691ebbb0ce9.png)


```suggestion
      # @note When using `pending` inside an example body using this method
      # hooks such as `before(:example) have already be run, this means that
      # a failure from the code in the `before` hook will prevent the example
      # from being considered as pending, as the example body wouldn't be
      # executed. If you need to consider hooks as pending as well you can use
      # the pending metadata as an alternative, e.g. `it "does something", pending: "message".`
```